Sources/SwiftFrameCore/Workers/TextRenderer.swift

```diff
@@ -97,3 +146,11 @@ final class TextRenderer {
     }
 
 }
+
+private func >=(lhs: CGSize, rhs: CGSize) -> Bool {
```

Comment:
Nice idea! Was surprised there wasn't a framework function for something similar.